allow passing options to Firefox webdriver

aculich opened the pull request:
This change matches what's in chrome.py#L28 to allow passing in options when
creating a new Browser() window. Also renamed variable from `firefox_options` to
just `options` to make it generic to match chrome.py.

```
from selenium.webdriver import Firefox, FirefoxOptions
opts = FirefoxOptions()
opts.add_argument("--some-argument")
browser = Browser('firefox', options=opts)
```

andrewsmedina commented on splinter/driver/webdriver/firefox.py:
```diff
@@ -40,7 +40,7 @@ def __init__(
         firefox_capabilities["marionette"] = True
 
-        firefox_options = Options()
+        options = Options() if options is None else options
```
It is necessary to add the option as parameter in the `__init__` method definition.

And we can simplify this code using `or` boolean operator:
```suggestion
        options = options or Options()
```

aculich replied:
Add this simplification to both the firefox and chrome drivers.

aculich commented:
Added it to __init__() so thanks for pointing that out.

Also, simplified it in both the firefox and chrome drivers to just `options = options or Options()`
